solanum-tuberosums/djangazon
# User can "soft delete" payment types

## Associated Ticket
[User Can Delete Payment Types](https://github.com/solanum-tuberosums/djangazon/issues/8)

## To Test:
- Make a payment type
- Delete a payment type
- Make sure payment type is not shown on My Account page AND on Completing an Order page.


## All Tests Pass
N/A
